probe/kubernetes/reporter.go

```diff
@@ -326,3 +341,18 @@ func (r *Reporter) podTopology(services []Service, replicaSets []ReplicaSet) (re
 	})
 	return pods, err
 }
+
+func addScalingControls(t report.Topology) {
```

Comment:
Can we do something similar to https://github.com/weaveworks/scope/blob/master/probe/docker/reporter.go#L53?